Public broadcasting of Latvia.js

```diff
@@ -0,0 +1,335 @@
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/raksts/')) {
+		return "newspaperArticle";
+	}
+	return false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "newspaperArticle") {
+		scrape(doc, url);
+	}
+}
+
+function formatDate(date) {
+	return date.toISOString().split('T')[0];
+}
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+
+	translator.setHandler('itemDone', function (obj, item) {
+		item.publicationTitle = ZU.xpathText(doc, '//section[@class="globalnav__item globalnav__item-sitename"]');
+		item.section = ZU.xpathText(doc, '//div[@class="article__info"]/section[3]');
+
+		item.creators = [];
+		
+		// Possible formats:
+		// Autori: Zanda Ozola-Balode (Panorāma), Linda Zalāne (Latvijas Radio Ziņu dienesta korespondente)
+		// Autori: Uģis Lībietis (Latvijas Radio ārzemju ziņu korespondents )
+		// Autori: Ludmila Glazunova
+		// Autori: LSM.lv ziņu redakcija
+		
+		var names = ZU.xpathText(doc, '//div[@class="article__info"]/section[@class="article__info__authors"]');
+		if (names) {
+			names = names.substring(names.indexOf(":") + 1);
+		
+			var authors = names.split(',');
+			
+			for (let author of authors) {
+				author = author.trim();
+				if (author === 'LSM.lv ziņu redakcija' || author === 'LETA' || author === 'BNS') {
+					item.creators.push({ lastName: author, creatorType: "author", fieldMode: true });
+				} else {
+					// Zanda Ozola-Balode (Panorāma)
+					if (author.indexOf('(') > 0) {
+						author = author.substring(0, author.indexOf('('));
+					}
+					item.creators.push(ZU.cleanAuthor(author, "author"));
+				}
+			}
+		}
+		
+		// Date string is of the format:
+		// Publicerat onsdag 15 november 2017 kl 00.11
+		// /html/body/main/section[8]/div/div[1]/section[1]/article/div[3]/section[1]
+		var dateString = ZU.xpathText(doc, '//div[@class="article__info"]/section[1]');
+		if (dateString.indexOf('Šodien') > 0) {
+			item.date = formatDate(new Date());
+		} else if (dateString.indexOf('Vakar') > 0) {
+			item.date = formatDate(new Date(new Date().setDate(new Date().getDate() - 1)));
+		} else {
+			// 14. jūlijs, 2014, 16:07
+			// 12. oktobris, 20:29
+			var dateParts = dateString.match(/(\d{1,2})\. (\w+), (\d{4}), .*/);
+			if (!dateParts) {
+				dateParts = dateString.match(/(\d{1,2})\. (\w+),.*/);
+				if (dateParts) {
+					dateParts[3] = new Date().getFullYear();
+				}
+			}
+			if (dateParts) {
+				var year = dateParts[3];
+				var months = {
+					janvāris: "01",
+					februāris: "02",
+					marts: "03",
+					aprīlis: "04",
+					maijs: "05",
+					jūnijs: "06",
+					jūlijs: "07",
+					augusts: "08",
+					septembris: "09",
+					oktobris: "10",
+					novembris: "11",
+					decembris: "12"
+				};
+				var month = months[dateParts[2]];
+				var day = dateParts[1];
+				item.date = year + "-" + month + "-" + day;
```

Comment:
It looks that you are re-implementing things which should already be covered with Zotero's date functionality. However, Latvian terms might be missing. @dstillman Should this go into the file https://github.com/zotero/zotero/blob/master/resource/schema/dateFormats.json instead? Then use `ZU.strToISO()`.